tests/src/test/scala/common/rest/WskRest.scala

```diff
@@ -0,0 +1,1434 @@
+    }
+    val resp = requestEntity(GET, entPath, paramMap)
+    val r = new RestResult(resp.status, getRespData(resp))
+    validateStatusCode(expectedExitCode, r.statusCode.intValue)
+    r
+  }
+
+  /**
+   * Gets entity from collection.
+   *
+   * @param name either a fully qualified name or a simple entity name
+   * @param expectedExitCode (optional) the expected exit code for the command
+   * if the code is anything but DONTCARE_EXIT, assert the code is as expected
+   */
+  override def get(name: String,
+                   expectedExitCode: Int = OK.intValue,
+                   summary: Boolean = false,
+                   fieldFilter: Option[String] = None,
+                   url: Option[Boolean] = None,
+                   save: Option[Boolean] = None,
+                   saveAs: Option[String] = None)(implicit wp: WskProps): RestResult = {
+    val (ns, entity) = getNamespaceActionName(name)
+    val entPath = Path(s"$basePath/namespaces/$ns/$noun/$entity")
+    val resp = requestEntity(GET, entPath)(wp)
+    val r = new RestResult(resp.status, getRespData(resp))
+    validateStatusCode(expectedExitCode, r.statusCode.intValue)
+    r
+  }
+}
+
+trait DeleteFromCollectionRest extends BaseDeleteFromCollection {
+  self: RunWskRestCmd =>
+
+  /**
+   * Deletes entity from collection.
+   *
+   * @param name either a fully qualified name or a simple entity name
+   * @param expectedExitCode (optional) the expected exit code for the command
+   * if the code is anything but DONTCARE_EXIT, assert the code is as expected
+   */
+  override def delete(entity: String, expectedExitCode: Int = OK.intValue)(implicit wp: WskProps): RestResult = {
+    val (ns, entityName) = getNamespaceActionName(entity)
+    val path = Path(s"$basePath/namespaces/$ns/$noun/$entityName")
+    val resp = requestEntity(DELETE, path)(wp)
+    val r = new RestResult(resp.status, getRespData(resp))
+    validateStatusCode(expectedExitCode, r.statusCode.intValue)
+    r
+  }
+
+  /**
+   * Deletes entity from collection but does not assert that the command succeeds.
+   * Use this if deleting an entity that may not exist and it is OK if it does not.
+   *
+   * @param name either a fully qualified name or a simple entity name
+   */
+  override def sanitize(name: String)(implicit wp: WskProps): RestResult = {
+    delete(name, DONTCARE_EXIT)
+  }
+}
+
+trait HasActivationRest extends HasActivation {
+
+  /**
+   * Extracts activation id from invoke (action or trigger) or activation get
+   */
+  override def extractActivationId(result: RunResult): Option[String] = {
+    extractActivationIdFromInvoke(result.asInstanceOf[RestResult])
+  }
+
+  /**
+   * Extracts activation id from 'wsk action invoke' or 'wsk trigger invoke'
+   */
+  private def extractActivationIdFromInvoke(result: RestResult): Option[String] = {
+    Try {
+      val activationID =
+        if ((result.statusCode == OK) || (result.statusCode == Accepted)) result.getField("activationId") else ""
```

Comment:
Here it looks like the an empty string returns `Some()`. I think `None` should be returned instead.

Reply:
Should only need this in the function: 
```
Try {
     if ((result.statusCode == OK) || (result.statusCode == Accepted)) Some(result.getField("activationId")) else None
}
```

Reply:
We should remove try {}, or it will report error.